Documentation/gettingstarted/clustermesh/affinity.rst

```diff
@@ -0,0 +1,146 @@
+.. _gs_clustermesh_service_affinity:
+
+****************
+Service Affinity
+****************
+
+This tutorial will guide you to enable service affinity across multiple
+Kubernetes clusters.
+
+Prerequisites
+#############
+
+You need to have a functioning Cluster Mesh with a Global Service, please
+follow the guide :ref:`gs_clustermesh` and :ref:`gs_clustermesh_services`
+to set it up.
+
+Enabling Global Service Affinity
+################################
+
+Load-balancing across multiple clusters might not be ideal in some cases.
+The annotation ``io.cilium/service-affinity: "local|remote|none"`` can be used
+to specify the ``preferred`` endpoint destination.
```

Comment:
Are the quotes around "preferred" needed, as it is not a keyword?

Reply:
good spot, it's not keyword, let me remove them.